flink-table/flink-table-planner/src/test/resources/jsonplan/testGetJsonPlan.out

```diff
@@ -1,76 +1,110 @@
 {
-   "flinkVersion":"",
-   "nodes":[
-      {
-         "class":"org.apache.flink.table.planner.plan.nodes.exec.stream.StreamExecTableSourceScan",
-         "scanTableSource":{
-            "identifier":{
-               "catalogName":"default_catalog",
-               "databaseName":"default_database",
-               "tableName":"MyTable"
+  "flinkVersion": "",
+  "nodes": [
+    {
+      "class": "org.apache.flink.table.planner.plan.nodes.exec.stream.StreamExecTableSourceScan",
+      "scanTableSource": {
+        "catalogTable": {
+          "identifier": "`default_catalog`.`default_database`.`MyTable`",
+          "catalogTable": {
+            "resolvedSchema": {
+              "columns": [
+                {
+                  "name": "a",
+                  "type": "physical",
+                  "dataType": "BIGINT"
+                },
+                {
+                  "name": "b",
+                  "type": "physical",
```

Comment:
let's still try to comply with `CatalogPropertiesUtil#serializeCatalogTable`. E.g. we can omit `"type": "physical"`

Reply:
> type computed is implicit if there is an expression, metadata with a metadata key

I rather prefer to avoid doing this. I agree that type can be default to physical if missing, but this inference doesn't sound right to me, as we don't know if at some point in the future we might want to have a supercolumn that has both an expression and a metadata field. I rather prefer to be conservative and serialize the type, just to be on the safe side. We can always remove this type in future.